qutebrowser/config/configdata.yml

```diff
@@ -259,6 +259,12 @@ content.default_encoding:
     The encoding must be a string describing an encoding such as _utf-8_,
     _iso-8859-1_, etc.
 
+content.desktop_fullscreen:
```

Comment:
This setting name seems kind of confusing to me, as a fullscreen being full-screen (duh 😆) is kind of the normal thing I think people would expect. What about naming it `windowed_fullscreen` instead and inverting the meaning?